src/StdChains.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "src/Vm.sol";
+
+/// @dev To add a new chain:
+///   1. Add it into the `Chains` struct, named using CamelCase.
+///   2. Initialize it in the `stdChains` declaration.
+///   3. Add a corresponding `else if` line into the constructor, remembering to check for both
+///      underscore and hyphenated versions of the chain-name when appropriate.
+abstract contract StdChains {
+    Vm private constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    struct Chain {
+        // The chain name, using underscores as the separator to match `foundry.toml` conventions.
+        string name;
+        // The chain's Chain ID.
+        uint256 chainId;
+        // A default RPC endpoint for this chain.
+        // NOTE: This default RPC URL is included for convenience to facilitate quick tests and
+        // experimentation. Do not use this RPC URL for production test suites, CI, or other heavy
+        // usage as you will be throttled and this is a disservice to others who need this endpoint.
+        string rpcUrl;
+    }
```

Comment:
this is pretty great, should just work. I don't think this will cause issues. but once this is ready we should add some tests to foundry as well